settings.py

```diff
@@ -90,9 +103,9 @@
 if os.path.isfile(ENV_ROOT / "env.json"):
     with open(ENV_ROOT / "env.json") as env_file:
         ENV_TOKENS = json.load(env_file)
-    TMP_GEN_DIR = ENV_TOKENS.get('TMP_GEN_DIR', '/tmp/certificates/')
-    QUEUE_NAME = ENV_TOKENS.get('QUEUE_NAME', 'test-pull')
-    QUEUE_URL = ENV_TOKENS.get('QUEUE_URL', 'https://stage-xqueue.edx.org')
+    TMP_GEN_DIR = ENV_TOKENS.get('TMP_GEN_DIR', TMP_GEN_DIR)
+    QUEUE_NAME = ENV_TOKENS.get('QUEUE_NAME', QUEUE_NAME)
```

Comment:
???

if env.json doesn't have anything it should take the default it gets passed, no?